ci/deploy-vmck

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -ex
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+export VAGRANT_DOTFILE_PATH=$(mktemp -d --tmpdir VAGRANT_DOTFILE_XXXXXXXX)
+export VAGRANT_DEFAULT_PROVIDER=vmck
+export VAGRANT_CHECKPOINT_DISABLE=true
+export VAGRANT_BOX_UPDATE_CHECK_DISABLE=true
+
+TIMEOUT_MIN=23
+RETRIES=3
+
+function print_section() {
+  set +x
+  echo
+  echo '-----------------------------------------'
+  echo "| $1"
+  echo '-----------------------------------------'
+  set -x
+}
+
+function vagrant_up() {
+  set +e
+  vagrant up --no-provision || echo "vagrant up failed, VM might still work"
+  echo "sudo shutdown +$TIMEOUT_MIN" | vagrant ssh
+  sshret=$?
+  if [ 0 -eq $sshret ]; then
+    return 0
+  else
+    vagrant destroy -f
+    return 1
+  fi
+}
+
+function retry_vagrant_up() {
+  for i in $(seq 1 $RETRIES); do
```

Comment:
I think retries are not helpful here. `vagrant up` should always work, and if it doesn't, that's a problem to fix. So let's simplify the code and just do `vagrant up`.